src/Resources/views/tailwind_2/_header.html.twig

```diff
@@ -10,7 +10,7 @@
             {% endif %}
         </div>
         <div class="mt-4 flex sm:mt-0 sm:ml-4 space-x-2">
-            {% if view is defined and view.definition.hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT')) %}
+            {% if table.exporters|length > 0 and table.option('definition').hasCapability(constant('araise\\CrudBundle\\Enums\\Page::EXPORT')) %}
```

Comment:
@dowitsch `table.option('definition')` can be `null`. Please add `null` check. 

--> https://github.com/araise-dev/TableBundle/blob/main/src/Table/Table.php#L127